hive-metastore/src/main/java/org/apache/iceberg/hive/MetastoreLock.java

```diff
@@ -74,7 +77,10 @@ class MetastoreLock implements HiveLock {
   private static final long HIVE_LOCK_CREATION_TIMEOUT_MS_DEFAULT = 3 * 60 * 1000; // 3 minutes
   private static final long HIVE_LOCK_CREATION_MIN_WAIT_MS_DEFAULT = 50; // 50 milliseconds
   private static final long HIVE_LOCK_CREATION_MAX_WAIT_MS_DEFAULT = 5 * 1000; // 5 seconds
-  private static final long HIVE_LOCK_HEARTBEAT_INTERVAL_MS_DEFAULT = 4 * 60 * 1000; // 4 minutes
+
+  @VisibleForTesting
+  static final long HIVE_LOCK_HEARTBEAT_INTERVAL_MS_DEFAULT = 60 * 1000; // 1 minutes
```

Comment:
HIVE_ACQUIRE_LOCK_TIMEOUT_MS_DEFAULT is 3 minutes, HIVE_LOCK_HEARTBEAT_INTERVAL_MS_DEFAULT 4 minutes is too long.